packages/components/src/internal/components/forms/QueryInfoForm.tsx

```diff
@@ -33,8 +36,12 @@ import { LoadingSpinner } from '../base/LoadingSpinner';
 import { QueryInfoQuantity } from './QueryInfoQuantity';
 import { QueryFormInputs, QueryFormInputsProps } from './QueryFormInputs';
 import { getFieldEnabledFieldName } from './utils';
+import { CommentTextArea } from './input/CommentTextArea';
+import { getAppHomeFolderPath } from '../../app/utils';
+import { ComponentsAPIWrapper, getDefaultAPIWrapper } from '../../APIWrapper';
 
 export interface QueryInfoFormProps extends Omit<QueryFormInputsProps, 'onFieldsEnabledChange'> {
+    api?: ComponentsAPIWrapper,
```

Comment:
These new props, `api` and `includeCommentField`, need to be excluded from the `...queryFormInputProps` in `render()` as they are not part of the `QueryFormInputsProps` interface.